bin/deployment/update-server-xml.py

```diff
@@ -12,7 +12,16 @@
 logging.basicConfig(level=logging.INFO, format="%(levelname)-7s %(message)s")
 logger = logging.getLogger('update_server_xml')
 
-
+SSL_CIPHERS = [
```

Comment:
We should also update cpsetup with the new config. I think the ideal would be to avoid duplication by extracting the cert/key generation and tomcat configuration in a separate python module, that can be used by both cpsetup and the deploy script, but that could be done as a separate task.

Reply:
No one uses cpsetup, and the delta that has grown between it and deploy is one that's been the subject of several discussions and at least one epic with several tasks attempting to fix it. Both cpsetup and cpdb are in dire need of love, but both, I feel, are outside the scope of the FIPS work.

Reply:
cpsetup is the only way of getting candlepin deployed after installing it as an rpm (which is something that we occasionally have to test), aside from actually doing all that work by hand. But I agree it is outside of the scope for FIPS (just that eventually we should update it)

Reply:
I wasn't aware of the RPM thing (since we do it ourselves so seldomly. That being the case, I'll do a very ugly update of cpsetup (since it looks pretty hack itself) sometime on Monday to basically just do the current changes without the legacy keystore bits.

In the future, we should probably set it up so that the RPM install script itself calls cpsetup so it's not necessary to do it after the fact -- especially if we plan to move to Spring Boot. (Also merge cpdb into cpsetup and convert the whole mess into a proper python package/module of sorts).